example/Demo.jsx

```diff
@@ -15,6 +15,14 @@ const terms = [
     term: 'demo',
     definition: 'a thing that breaks on presentation',
   },
+  {
+    term: 'exogenous',
+    definition: 'relating to or developing from external factors',
+  },
+  {
+    term: 'endogenous',
+    definition: 'having an internal cause or origin',
+  },
```

Comment:
Added so the glossary terms actually get tooltips in the demo here: https://markdown-pr-553.herokuapp.com/#features